packages/block-directory/src/components/downloadable-block-list-item/style.scss

```diff
@@ -0,0 +1,48 @@
+.block-directory-downloadable-block-list-item {
+	width: 100%;
+	padding: 0;
+	margin: 0 0 12px;
+	display: flex;
+	flex-direction: row;
+	font-size: $default-font-size;
+	color: $dark-gray-700;
+	align-items: flex-start;
+	justify-content: center;
+	background: transparent;
+	word-break: break-word;
+	border-radius: $radius-round-rectangle;
+	border: $border-width solid $light-gray-500;
+	transition: all 0.05s ease-in-out;
+	@include reduce-motion("transition");
+	position: relative;
+	text-align: left;
```

Comment:
There's a subtle little thing here where the gray box at the bottom of this rounded box gets ever so slightly clipped. It is more visible if you increase the border radius:

<img width="501" alt="Screenshot 2019-09-13 at 14 04 12" src="https://user-images.githubusercontent.com/1204802/64884768-7b7bdc80-d630-11e9-8f74-e72c5728e770.png">

You can either fix this by adding `overflow: hidden;` to this rule, so the border radius clips the corners of the bottom box:

<img width="393" alt="Screenshot 2019-09-13 at 14 12 40" src="https://user-images.githubusercontent.com/1204802/64884801-8fbfd980-d630-11e9-8ea1-414cf8fd6023.png">

Or if you're concerned about side effects of changing the overflow properties, you could potentially change the radius of the bottom box to match, something along the lines of:

☝️note the border radius in the above screenshots is exaggerated to show the effect.

```
.block-directory-downloadable-block-list-item__footer {
	...
	border-radius: 0 0 3px 3px;
}
```

However I think it would be preferable if you could go the overflow route.

Reply:
Well spotted! Now fixed using overflow: hidden 👍